src/pages/tags/[...slug].astro

```diff
@@ -1,16 +1,16 @@
 ---
 import { getCollection } from "astro:content";
 
-import Aside from "@/components/Aside.astro";
+import aside from "@/components/aside.astro";
```

Comment:
Imported components use `PascalCase`, while the filenames across the project (besides the documents) use `lower-kebab-case`.

I intend to find/create a way to automate case maintenance so we don't have to think about it as much.

`import Aside from "@/components/aside.astro";`